lib/licensee/project_files/license_file.rb

```diff
@@ -3,6 +3,32 @@ class Project
     class LicenseFile < Licensee::Project::File
       include Licensee::ContentHelper
 
+      # List of extensions to give preference to
+      EXTENSIONS = %w(md markdown txt).freeze
+      EXTENSIONS_REGEX = Regexp.union(EXTENSIONS)
+
+      # Regex to match any extension
+      ANY_EXTENSION_REGEX = /[^.]+/
```

Comment:
I'd rename `EXTENSONS_REGEX` to `FORMAT_EXTENSION_REGEX` or `TEXT_EXTENSION_REGEX`, to make the composition below clearer and more consistent.